openmp/libompd/gdb-plugin/ompdModule.c

```diff
@@ -574,7 +574,7 @@ ompd_rc_t _read_string (
 	}
 	Py_ssize_t retSize;
 	const char* strbuffer = PyUnicode_AsUTF8AndSize(retString, &retSize);
-	if(retSize >= nbytes) {
+	if((unsigned int)retSize >= nbytes) {
```

Comment:
> The returned buffer always has an extra null byte appended (not included in size), regardless of whether there are any other null code points.
I think this should be:
```suggestion
	if((ompd_size_t)retSize+1 >= nbytes) {
```